css-positioning/relative.css

```diff
@@ -0,0 +1,46 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
+  top: 20px;
+  right: 55px;
+}
+
+#charmander {
+  /* your code here */
+  position: relative;
+  top: 40px;
+  right: 55px;
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
+  right: 55px;
+  top: 75px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
+  right: 55px;
+  top: 160px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
+  right: 55px;
+  top: 300px;
+}
```

Comment:
What ruleset do we have that affects all of our images? Can we add `position: relative` there instead of adding it to each individual ruleset?